core/primitives-core/src/version.rs

```diff
@@ -207,10 +207,7 @@ impl ProtocolFeature {
             | ProtocolFeature::TestnetFewerBlockProducers
             | ProtocolFeature::SimpleNightshadeV2 => 64,
             ProtocolFeature::SimpleNightshadeV3 => 65,
-            ProtocolFeature::DecreaseFunctionCallBaseCost => 66,
-
-            // Nightly features which should be tested for compatibility with resharding
-            ProtocolFeature::YieldExecution => 78,
+            ProtocolFeature::DecreaseFunctionCallBaseCost | ProtocolFeature::YieldExecution => 66,
```

Comment:
For my own understanding, why are we reusing version 66 instead of bumping to 67? Wasn't DecreaseFunctionCallBaseCost already released?

Reply:
ah yeah, you're right, that's what polina said too.